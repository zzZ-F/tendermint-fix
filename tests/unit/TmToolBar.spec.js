import { mount } from "@vue/test-utils"
import { TmToolBar } from "@/index"

describe("TmToolBar", () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(TmToolBar)
  })

  it("has the expected html structure", () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
