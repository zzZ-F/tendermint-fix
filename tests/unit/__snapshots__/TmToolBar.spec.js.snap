// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmToolBar has the expected html structure 1`] = `
<div class="tm-tool-bar">
  <div class="tm-tool-bar-container">
    <div class="main"></div>
  </div>
</div>
`;
